packages/now-cli/package.json

```diff
@@ -61,12 +61,12 @@
     "node": ">= 8.11"
   },
   "devDependencies": {
-    "@now/build-utils": "0.9.14-canary.2",
-    "@now/go": "latest",
-    "@now/next": "latest",
-    "@now/node": "latest",
-    "@now/routing-utils": "1.2.3-canary.1",
-    "@now/static-build": "latest",
+    "@now/build-utils": "./packages/now-build-utils",
+    "@now/go": "./packages/now-go",
+    "@now/next": "./packages/now-next",
+    "@now/node": "./packages/now-node",
+    "@now/routing-utils": "./packages/now-routing-utils",
+    "@now/static-build": "./packages/now-static-build",
```

Comment:
@TooTallNate What do you think of this?

I think we should delete the devDependencies here and instead read from the top-level `node_modules` which should be symlinks.

Reply:
Updated this with master state for now 🙏🏻